### DISCUSSION TITLE
Add a discord command to respond with time to next task runs

### DISCUSSION THREAD

**Discussion by TheHoboArk** [gameserverapp/Platform#390] [Suggestions]
There are multiple scheduled tasks that need to run throughout the day (for example, wipe dinos, or something similar), and it would be helpful to be able to send a command in discord, and the bot would respond with
`3hrs 24min until task runs again`

Or something similar that is editable my the server owners.

As a sub-item, allowing a new variable to be added to the "Send Discord Message" task, called {timetonexttaskrun} or something similar, could (while not in a responsive manner), be used to send periodic updates, instead of answering.

**Comment by mvaessen** [comment]
<img width="458" height="266" alt="Screenshot 2026-02-26 at 20 15 20" src="https://github.com/user-attachments/assets/b4f74bf0-38cd-4698-9e91-2c8e1603c244" />
<img width="379" height="90" alt="Screenshot 2026-02-26 at 20 15 40" src="https://github.com/user-attachments/assets/41da75bb-99f6-4949-95d9-275c91f57b23" />

This is available tomorrow. Comes with a variety of options:
```
{task.115.last_run_datetime}
{task.115.next_run_datetime}
{task.115.minutes_until_run}
{task.115.hours_until_run}
{task.115.days_until_run}
{task.115.diff_for_humans}
```

These variables can be used pretty much anywhere
